src/components/input-editable/input-editable.tsx

```diff
@@ -103,50 +117,50 @@ export class InputEditable {
   render() {
     return (
       <Host>
-        <div class="input__editable">
-          <div
-            class={{ 'input__editable--static': true, 'input__editable--hidden': this.isEditing }}
-            onClick={this.handleEditing}
+          <div class="input__editable">
+          <div class={{ 'input__editable--static': true, 
+                        'input__editable--hidden': this.isEditing }}
+               onClick={this.handleEditing}
           >
             <bds-typo
               tag="span"
               part="input__editable--static__typo"
               class="input__editable--static__typo"
-              variant="fs-24"
+              variant={this.variant}
```

Comment:
These font variants are just two. Default (24) and for 100% width (16). Can't be chanceable.